manifests/init.pp

```diff
@@ -472,6 +475,7 @@
   $default_ssl_crl_path                                                 = undef,
   $default_ssl_crl                                                      = undef,
   $default_ssl_crl_check                                                = undef,
+  $default_ssl_reload_on_change                                         = false,
```

Comment:
could you please add a datatype for the new parameter?
```suggestion
  Boolean $default_ssl_reload_on_change                                 = false,
```

Reply:
@bastelfreak Done.